docs/deployment/manifest-format.md

```diff
@@ -358,7 +457,17 @@ Example manifest:
       "connectionString": "myredis:6379"
     },
     "redis2": {
-      "type": "redis.v0"
+      "type": "container.v0",
+      "image": "redis:latest",
+      "bindings": {
+        "tcp": {
+          "scheme": "tcp",
+          "protocol": "tcp",
+          "transport": "tcp",
+          "containerPort": 6379
+        }
+      },
+      "connectionString": "{rediscontainer.bindings.tcp.host}:{rediscontainer.bindings.tcp.port}"
```

Comment:
Should be: `{redis2.bindings.tcp.host}:{redis2.bindings.tcp.port}`

Reply:
Done